src/content/docs/cypher/query-clauses/call.md

```diff
@@ -97,19 +100,20 @@ CALL show_tables() RETURN *;
 ```
 Output:
 ```
---------------------------------------------
-| name        | type | comment             |
---------------------------------------------
-| gf          | RDF  |                     |
---------------------------------------------
-| gf_TRIPLES  | REL  |                     |
---------------------------------------------
-| gf_RESOURCE | NODE |                     |
---------------------------------------------
-| person      | NODE | person info         |
---------------------------------------------
-| knows       | REL  | person knows person |
---------------------------------------------
+-------------------------------------------------------
```

Comment:
Again, please use an example database and output this table using the latest version of the CLI. Also, remove the RDF data type from the first row as we're disabling RDF from the next release.